cli/command/stack/list.go

```diff
@@ -86,27 +75,3 @@ func runList(cmd *cobra.Command, dockerCli command.Cli, opts options.List) error
 	})
 	return formatter.StackWrite(stackCtx, stacks)
 }
-
-func getStacks(dockerCli command.Cli, opts options.List, kopts kubernetes.Options) ([]*formatter.Stack, error) {
-	kli, err := kubernetes.WrapCli(dockerCli, kopts)
-	if err != nil {
-		return nil, err
-	}
-	ss, err := kubernetes.GetStacks(kli, opts)
-	if err != nil {
-		return nil, err
-	}
-	return ss, nil
-}
-
-func removeDuplicates(namespaces []string) []string {
```

Comment:
I just noticed that this commit https://github.com/docker/cli/pull/1034/commits/691eeff5a5db87be3fe5e0f1ca852024edf08f1c is refactoring code that wasn't even merged yet: https://github.com/docker/cli/pull/1031/commits/1f9c9767573c7b8876895d27a9d9b996216e8203 (https://github.com/docker/cli/pull/1031). To reduce code-churn, could this refactor be done in the commit that adds it?

Reply:
Yes, good point.